SIP/sip-48.md

```diff
@@ -1,74 +1,120 @@
+
+Signum will provide the following STLD by default:
+
+- blockhain
+- coin
+- crypto
+- dao
+- decentral
+- dex
+- free
+- nft
+- p2p
+- sig
+- signa
+- sns
+- w3
+- wallet
+- web3
+- x
+- y
+- z
 
+Aliases created without an STLD (e.g. all Aliases created before this SIP activates) have the .signum STLD for free. 
 
-## Specification
-The following changes should be done with an upcoming hard fork to introduce the renewal process for aliases.
 
-### Database changes
-On the database level, the alias table will get a new field renewal_time which is an INT field and contains the timestamp until it stocks the Alias for the current owner. With the introduction of the needed hard fork, all aliases will get a DateTime of one year after the planned hard fork. In this case, all active users have enough time to check their current alias ownerships and can create a renewal subscription if desired.
+Example:
 
-### Grace Period
-For all existing Aliases before the hard fork, the node sets a renewal time from plus one year (based on the hard fork block).
+Suppose a user creates an Alias without an STLD selection like “*Spaceship*”. In that case, this Alias gets registered for “*Spaceship*” and “*Spaceship.signum*”
 
-### Node changes
-**Transaction Set Alias**
 
-If a user sends a setAlias transaction, it will execute the following DateTime handling for the Alias:
- - If the creation is before the block of the hard fork, it sets no DateTime on `renewal_time`.
- - If the creation is after the hard fork and no DateTime exits for the Alias, it sets a new DateTime with DateTime plus 24 hours on `renewal_time`. The same applies to a DateTime which is in the past or shorter than 24 hours in the future.
- - If the creation is after the hard fork, but the DateTime is longer than 24 hours in the future, no DateTime adjustment will happen on `renewal_time`.
- - If the setAlias leads to a new Alias, it sets the renewal_time with a value of 24 hours in the future.
+Suppose a user created an Alias “*Spaceship*” with the STLD, for example, “*crypto*”. In that case, this Alias is now only registered for "*Spaceship.crypto*”.
 
-In addition, if it executes the setAlias transaction for an existing Alias and the creator of the setAlias transaction is not the current owner, the transaction - which leads to an automated owner transfer - will only be executed if the renewal timestamp of the Alias has already expired.
 
-**Handling of Alias offers(Sale)**
 
-If a `setAlias` transaction leads to an owner change, the node needs to check for ongoing Sales (public/private). If a sale exists, the node code removes the Sale, setting the `latest` field for this offer from 1 to 0.
+### Custom STLD
+Besides the default STLDs, any Signum account can create a new STLD on the Signum chain. This way, specific use- or business cases can build their namespaces and benefit from those. As Aliases have a quarterly renewal cost, the chain will send those payments to the owner of the STLD.
```

Comment:
This way, specific use or business cases can build their namespaces and benefit from those.
Delete the `-` after use